metadata-service/factories/src/main/resources/application.yml

```diff
@@ -289,6 +289,7 @@ featureFlags:
   showBrowseV2: ${SHOW_BROWSE_V2:true} # Enables showing the browse v2 sidebar experience.
   preProcessHooks:
     uiEnabled: ${PRE_PROCESS_HOOKS_UI_ENABLED:true} # Circumvents Kafka for processing index updates for UI changes sourced from GraphQL to avoid processing delays
+  showAcrylInfo: ${SHOW_ACRYL_INFO:true} # Show different CTAs within DataHub around moving to Managed DataHub. Set to true for the demo site.
```

Comment:
Default to false here probably?

Reply:
oh man amazing call haha woops! i forgot to switch it back